src/smclarify/bias/report.py

```diff
@@ -99,6 +99,26 @@ def toJson(self):
         return json.loads(json.dumps(self, default=lambda o: o.__dict__), object_hook=inf_as_str)
 
 
+class ModelPerformanceReport:
+    """Model Performance Report with label name, list of MetricResult objects, confusion_matrix, binary_confusion_matrix"""
+
+    def __init__(
+        self,
+        label_name: str,
+        metrics: List[MetricResult],
+        binary_confusion_matrix: List[float],
+        confusion_matrix: Optional[Dict] = None,
+    ):
+        self.label = label_name
+        self.model_performance_metrics = metrics
+        self.binary_confusion_matrix = binary_confusion_matrix
+        if confusion_matrix:
+            self.confusion_matrix = confusion_matrix
+
+    def toJson(self):
+        return json.loads(json.dumps(self, default=lambda o: o.__dict__), object_hook=inf_as_str)
```

Comment:
is this returning a dict? if you load it back?

Reply:
It's returning a dict when I load it back yes.